### ISSUE TITLE
CGAL testsuite is broken with Qt 5.10

### ISSUE THREAD

**Issue by lrineau** [CGAL/cgal#2754]
In the latest test results, all the platforms using Qt 5.10 (currently ArchLinux and Fedora-rawhide) have compilation errors with all demos:
https://cgal.geometryfactory.com/CGAL/testsuite/results-4.12-Ic-152.shtml#AABB_tree_Demo

Maxime, can you try to find out why, and then hopefully find a solution?

**Comment by maxGimeno** [comment]
Apparently, Qt5.10 is not able to generate mocs  on the testsuite machines, but it works perfectly on mine. Maybe it's the packaged version that is bugged.

**Comment by afabri** [comment]
@lrineau Do you confirm that this is critical and must be adressed?

**Comment by lrineau** [comment]
Unfortunately, that is a real issue for our testsuite process: we can no longer test our demos on recents Linux versions. But, as far as Fedora is concerned, that is because of a new security policy, that is not yet well tuned.

I have filled a bug, two weeks ago:
https://bugzilla.redhat.com/show_bug.cgi?id=1540634
One of the answer from a Redhat Security Engineer was:|
> The map permission is a new permission recognized by kernel and policy. We are still in a process of identification of which domains need that permission when accessing various objects on the file-system.
> 
> More info can be found at:
> * https://patchwork.kernel.org/patch/9713531/
> 
> Could you tell us, which policy version is installed on your machine?

We cannot do anything but wait for a fix, or try to find a workaround. In my opinion that does not block CGAL-4.12-beta1, because in the work case I can only mean that a *demo* does not compile properly: tests and examples are correctly compiled and tested. I have changed the milestone...

**Comment by sloriot** [comment]
According to the ticket, a fix was pushed upstream, we should probably give it a new try. Shall @maxGimeno prepare a PR to reenable QT5 testing on the affected platforms?

**Comment by lrineau** [comment]
The affected platforms cannot easily upgrade both Docker, libseccomp, and the kernel. On cgal.geometryfactory.com, I have tried to upgrade the kernel, and Docker, and then when I saw that I had to upgrade libseccomp as well, I reverted my previous changes. Too risky in my opinion.

So we test Qt5 inside Docker containers on newer host machine, like our desktops at GF's office: renoir, bonnard, and friedrich.
